src/state/pools/hooks.ts

```diff
@@ -86,6 +90,29 @@ export const usePoolsPageFetch = () => {
   }, [dispatch])
 }
 
+export const useFetchIfo = () => {
+  const { account } = useWeb3React()
+  const dispatch = useAppDispatch()
+
+  useFastRefreshEffect(() => {
+    batch(() => {
+      dispatch(fetchCakeVaultPublicData())
+      dispatch(fetchIfoPublicDataAsync())
+      if (account) {
+        dispatch(fetchPoolsUserDataAsync(account))
+        dispatch(fetchCakeVaultUserData({ account }))
+        dispatch(fetchUserIfoCreditDataAsync(account))
+      }
+    })
+  }, [dispatch, account])
+
+  useEffect(() => {
+    batch(() => {
```

Comment:
Batch can be removed here

Reply:
`usePoolsPageFetch`  need to remove `batch` ?

https://github.com/ChefMomota/pancake-frontend/blob/bf14aa10ca7ba19b2ce27224a2a9737e30b2ee12/src/state/pools/hooks.ts#L71-L91

Reply:
Yes, batch is only needed when there is multiple dispatches to reduce renders